spec/jobs/update_s3_inventory_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+require "file_store/s3_store"
+
+describe Jobs::UpdateS3Inventory do
+  before do
+    SiteSetting.enable_s3_uploads = true
+    SiteSetting.s3_access_key_id = "abc"
+    SiteSetting.s3_secret_access_key = "def"
+    SiteSetting.enable_s3_inventory = true
+
+    store = FileStore::S3Store.new
+    @client = Aws::S3::Client.new(stub_responses: true)
+    store.s3_helper.stubs(:s3_client).returns(@client)
+    Discourse.stubs(:store).returns(store)
+  end
+
+  it "picks gravatar if system avatar is picked and gravatar was just downloaded" do
+    @client.expects(:put_bucket_policy)
```

Comment:
I don't think we need this line too.

Reply:
same as [above](https://github.com/discourse/discourse/pull/6867#discussion_r252837340).